payloads/external/edk2/Makefile

```diff
@@ -373,6 +373,20 @@ ifneq ($(CONFIG_CPU_MAX_TEMPERATURE),)
 BUILD_STR += --pcd gDasharoSystemFeaturesTokenSpaceGuid.PcdCpuMaxTemperature=$(CONFIG_CPU_MAX_TEMPERATURE)
 endif
 endif
+# gEfiSecurityPkgTokenSpaceGuid.PcdTpm2HashMask = 0x0000001f
+# The tested options are mutually exclusive (there is no option for SM3 256).
```

Comment:
Could you elaborate in the comment that this is design decision made by coreboot, preferably with link to either change that caused it or coreboot (not Dasharo) issue with discussion?

This WILL impact TrenchBoot...

Reply:
> Could you elaborate in the comment that this is design decision made by coreboot, preferably with link to either change that caused it or coreboot (not Dasharo) issue with discussion?

https://github.com/Dasharo/coreboot/compare/8a68c4eb9a6e7bc00f9234f6614b8220bf32038e..1cb3194d58b43e2c36d440992fec2914df3e67df

> This WILL impact TrenchBoot...

https://github.com/Dasharo/dasharo-issues/issues/982#issuecomment-2289055801